### ISSUE TITLE
Convert lattice.py docstrings to NumPy style

### ISSUE THREAD

**Issue by pavoljuhas** [diffpy/diffpy.structure#12]
Please convert docstrings of the Lattice class in the python3 branch to the NumPy style.  For this you can follow a template of partial conversion of the `Atom` class docs in e186ab6f0e38909bd8f5bece8f0934e8e4f83a42.

To start working on this

- [ ] create a new topic branch say `py3-lattice-docs` from the tip of the [python3 branch](https://github.com/diffpy/diffpy.structure/tree/python3).
- [ ] setup a development Anaconda environment `dev35` with Python 3.5 and sphinx documentation tool.  Make sure your installation passes tests and you can build the manual:
```sh
# one time only - run this from the base directory of this repo
conda create -n dev35 python=3.5 sphinx numpy
source activate dev35
python setup.py develop

# make sure package tests pass
python -m diffpy.structure.tests.run

# compile and review the manual, check just the `Lattice` link.
make -C doc/manual html
open doc/manual/build/html/index.html
```

- [ ] edit docstrings in [lattice.py](https://github.com/diffpy/diffpy.structure/blob/python3/src/diffpy/structure/lattice.py)  to get them all to the NumPy style formatting and make the "Lattice" class documentation look well in html document.

For guidance see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy.

- [ ] finally, create a pull request towards the `python3` branch in diffpy/diffpy.structure.

**Comment by chiahaoliu** [comment]
@pavoljuhas should we close this issue?

**Comment by pavoljuhas** [comment]
Sure.  Resolved via f1a2d1cfe482a9d755cad0ab8373013a77188203.
